Makefile

```diff
@@ -96,7 +96,7 @@ docker-push:
 
 ## Agent Docker Zone
 # Only use it for local testing, that's it
-AGENT_ES_HOSTS?=http://127.0.0.1:9200
+AGENT_ES_HOSTS?=[http://127.0.0.1:9200]
```

Comment:
my "make" knowledge is weak - is this making it an array? I usually assume that make syntax is the same as bash (I know it's not), but this would just be a string that starts with `[` and ends with `]`

Reply:
It's still just a string, but when it's passed inside the docker image it is embedded into a yaml file as is, and yaml recognises this as an array :)

So multi-level magic!